### ISSUE TITLE
Drop our bootstrap code for telemetry pings

### ISSUE THREAD

**Issue by ghost** [mozilla/testpilot#3179]
We ping for overall metrics as well as to be a proxy for other web extensions.  We should drop the bootstrap parts of this, but we need to finish https://bugzilla.mozilla.org/show_bug.cgi?id=1280234 first

**Comment by lmorchard** [comment]
Yeah, this should be replaced with a WebExtension Experiment API that offers Telemetry access. (Or an actual in-product API for Telemetry in WebExtensions, if such a thing lands first)
